README.md

```diff
@@ -26,12 +26,27 @@ If you're on Windows, use the open source telnet client [PuTTY](https://www.chia
 
 ## How to run it locally
 
-With a modern node installation available, just start it with
+### With a modern node installation available, just start it with
```

Comment:
Unrelated:
This shouldn't be a headline. Otherwise, the headline above has no paragraph.